maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -185,7 +188,13 @@ private void loadProjectProperties(Builder builder, MavenProject project) throws
 
 		// Merge in current project properties
 		File baseDir = project.getBasedir();
-		File bndFile = new File(baseDir, Project.BNDFILE);
+		File bndFile;
+		if(bndLocation != null) {
+			bndFile = new File(bndLocation);
```

Comment:
This won't work. `loadProjectProperties` is recursively called to load the bnd file of the parent project. So the configuration of the location of the bnd file in _this_ project may have no relation to the location of the bnd file in all the parent projects.

Note: It is also possible that a parent project may have a bnd file yet itself not be configured for the bnd-maven-plugin. In this way, they can provide common bnd settings for child projects.